src/pages/instances/InstanceDetailActions.tsx

```diff
@@ -70,7 +72,9 @@ const InstanceDetailActions: FC<Props> = ({ instance, project, isLoading }) => {
           )}
         </ContextualMenu>
       ) : (
-        <>{menuElements}</>
+        <div className="p-segmented-control" key="controlButtons">
```

Comment:
why is the `controlButtons` key needed here?

Reply:
right, we don't need the key here. removing it.